site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md

```diff
@@ -0,0 +1,117 @@
+
+Load balancer inventory collection has been added to Azure EMS refresh, as well as support for external proxy. The issue where inventory collection would bypass the proxy when collecting templates has been fixed. You can also make use of provider-specific logging with Azure to make debugging much easier.
+
+A new cache module is deployed in SmartState for Microsoft Azure, which reduces the average read requests from &gt;7000 to &lt;1000. Scan times have also been reduced from &gt;20 min to &lt;5 min!
+
+Finally, we noticed that Azure was missing pre- and post-provision methods in automate modeling, so Azure method points have been added for cloud provisioning in Automate.
+
+[![Azure PrePosition](/assets/images/blog/Euwe_GA_azure_preposition.png)](/assets/images/blog/Euwe_GA_azure_preposition.png)
+
+Many corresponding enhancements are applied to Google Compute Engine, including metrics, external proxy support, load balancer refresh, and pre/post-provisioning methods and schema changes in the method class for Automate.
+
+[![Google PostPosition](/assets/images/blog/Euwe_GA_google_postposition_thumb.png)](/assets/images/blog/Euwe_GA_google_postposition.png)
+
+During provisioning, the deprecated images are now hidden. Support for parsing Google health checks is added for load balancer refresh.
+
+In GCE, there are [preemptible instances](https://cloud.google.com/compute/docs/instances/preemptible). With Google’s contributions, we have inventory collection and provisioning of these preemptible VMs. Support for instance retirement is improved too.
+
+## OpenStack Improvements
+
+Working with the OpenStack team, we bring you a long list of improvements for the OpenStack cloud provider. To start off, collection of Host Aggregate inventory allows the OpenStack scheduler to enable advanced scheduling. The mapping of external cloud tenants to ManageIQ tenants is achieved with a post-refresh hook on the OpenStack provider which synchronizes after refresh.
+
+Creating, updating and deleting cloud tenants and host aggregates can now be carried out through the user interface from the Clouds menu under Compute.
+
+[![Add New Cloud Tenant](/assets/images/blog/Euwe_GA_add_cloud_tenant_thumb.png)](/assets/images/blog/Euwe_GA_add_cloud_tenant.png)
+
+Block storage service is introduced with Cinder backup and restore actions. Snapshot functionality has been added for OpenStack cloud volumes and VMs, and you can create and remove snapshots from the UI. New topology view for OpenStack cloud provider shows the availability zones, tenants and instances.
+
+Flavors are mapped to cloud tenants during OpenStack cloud refresh so that they are grouped correctly. Previously Floating IPs can be assigned only during provisioning, however with this release, associating/disassociating Floating IPs can be controlled from the configuration menu after provisioning.
```

Comment:
Previously, floating IPs could only be assigned during provisioning; however, with this release .....